test/mock/constants.ts

```diff
@@ -0,0 +1,4 @@
+export const RABBIT_URL = 'amqp://admin:admin@127.0.0.1:5672';
+export const RABBIT_EVENT_PEOPLE_APP_NAME = 'event_people_node';
+export const RABBIT_EVENT_PEOPLE_VHOST = 'event_people';
```

Comment:
In this case there're two options, exclude this files and import directly from prod costs, or create other consts just used for mock, like:

```
RABBIT_URL='any_url:any@1.2.4.5:6789
```